contrib/win32/win32compat/termio.c

```diff
@@ -260,6 +260,18 @@ int
 syncio_close(struct w32_io* pio)
 {
 	debug4("syncio_close - pio:%p", pio);
+
+	/* Flush descriptor.*/
+	if (pio->write_details.pending) {
+		WaitForSingleObject(pio->write_overlapped.hEvent, INFINITE);
+
+		/* drain queued APCs */
+		SleepEx(0, TRUE);
+	}
+
+	// This call is likely racy, there's no guarantee
+	// that a thread has begun IO operations when it's called.
+	// Why stop io operations when we're going to close it anyhow?
 	CancelIoEx(WINHANDLE(pio), NULL);
```

Comment:
Hi, this comment is mostly here because I'm unsure about what CancelIoEx is supposed to be doing here. If it is needed, I think its not doing what its supposed to be doing because of the possible race condition. If it is not needed, then I believe that this PR can be simplified to just removing it.

Reply:
We will take the change as it is. You can delete this comment.